### ISSUE TITLE
movies (Trac #361)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#360]
Making animations with the glshovel should be a little easier.  Hitting record should give you an animation as opposed to a series of jpegs.

Migrated from https://code.icecube.wisc.edu/ticket/361
```json
{
    "status": "closed", 
    "changetime": "2019-01-11T22:54:35", 
    "description": "Making animations with the glshovel should be a little easier.  Hitting record should give you an animation as opposed to a series of jpegs.", 
    "reporter": "olivas", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1547247275950644", 
    "component": "glshovel", 
    "summary": "movies", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2012-02-29T06:42:01", 
    "milestone": "", 
    "owner": "olivas", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20120229T11:51:06: blaufuss commented:

> And please make the "record" button look a whole lot less like a "STOP" button.
>

**Comment by icecube-trac** [comment]
Trac update at 20120301T00:04:03: 
* olivas changed status from "new" to "closed"
* olivas commented:

> Fixed in revision [85795].


* olivas changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190111T22:54:35: nwhitehorn commented:

> In changeset 169743:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="169743"
> fix libzma/libarchive confusion
> 
> closes #360
> ```
